pkg/common-controller/snapshot_update_test.go

```diff
@@ -327,6 +322,34 @@ func TestSync(t *testing.T) {
 			expectSuccess:    true,
 			test:             testSyncSnapshot,
 		},
+		{
```

Comment:
Thinking of some more test cases:
1. vs object status == nil, vs content object status error != nil -> error expected in volume snapshot object error status.
2. vs object status == nil,  vs content object status error == nil, no change expected in volume snapshot object error status.

Reply:
Added two new test cases.
Test case 2 behaves like this: When both statuses are nil, updateSnapshotStatus will create an Status and set boundContentName and set readyToUse to false.